config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -768,64 +683,6 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-containerd
     testgrid-tab-name: e2e-gci-reboot
-- interval: 2h
-  name: ci-cri-containerd-e2e-gci-gce-sd-logging
```

Comment:
etc.

basically while I agree these have been failing for way too long, I think I would save the commit touching this file for a "remove perma-failing" PR, not a "remove out of support branches" PR

Reply:
Totally.  Am separating.